host/cli/log.go

```diff
@@ -1,10 +1,13 @@
 package cli
 
 import (
+	"bufio"
+	"container/ring"
```

Comment:
The `container/ring` package is overkill (uses interfaces and pointers a lot) and no one really uses it. All you need is a slice and a cursor (have a look at `logaggregator/ring`, though don't use it, I'm redoing the package).